### PR TITLE
feat: use relative entity  path in schema actions

### PR DESCRIPTION
**Pull request by Raubzeug**
### Test Status: <span style="color: green;">✅ PASSED</span>
📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/1366/)

| Total | Passed | Failed | Flaky | Skipped |
|:-----:|:------:|:------:|:-----:|:-------:|
| 124 | 124 | 0 | 0 | 0 |

### Bundle Size: ✅
Current: 79.08 MB | Main: 79.08 MB
Diff: +0.00 MB (0.00%)

✅ Bundle size unchanged.

<details>
<summary>ℹ️ CI Information</summary>

- Test recordings for failed tests are available in the full report.
- Bundle size is measured for the entire 'dist' directory.
- 📊 indicates links to detailed reports.
- 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
</details>

### CODE REVIEW

File: src/containers/Tenant/ObjectSummary/transformPath.ts [hunk 1]
```diff
@@ -14,3 +19,19 @@ export function transformPath(path: string, dbName: string): string {
 
     return result;
 }
+
+export function isDomain(path: string, type?: EPathType) {
```
Comment (astandrik) [inline src/containers/Tenant/ObjectSummary/transformPath.ts:23]: Some tests would be very great

Comment (astandrik) [inline src/containers/Tenant/ObjectSummary/transformPath.ts:23]: export function isDomain(path: string, type?: EPathType): boolean {
    if (type !== EPathType.EPathTypeDir) {
        return false;
    }

    return path.split('/').length === 2;
}

Comment (astandrik) [inline src/containers/Tenant/ObjectSummary/transformPath.ts:23]: import { isDomain } from './path-utils';
import { EPathType } from './types';

describe('isDomain', () => {
    it('should return true for valid domain paths', () => {
        expect(isDomain('domain/', EPathType.EPathTypeDir)).toBe(true);
        expect(isDomain('another-domain/', EPathType.EPathTypeDir)).toBe(true);
    });

    it('should return false for non-directory paths', () => {
        expect(isDomain('domain/', EPathType.EPathTypeFile)).toBe(false);
        expect(isDomain('domain/', undefined)).toBe(false);
    });

    it('should return false for paths without trailing slash', () => {
        expect(isDomain('domain', EPathType.EPathTypeDir)).toBe(false);
    });

    it('should return false for paths with multiple slashes', () => {
        expect(isDomain('domain/subdomain/', EPathType.EPathTypeDir)).toBe(false);
        expect(isDomain('/domain/', EPathType.EPathTypeDir)).toBe(false);
    });

    it('should return false for empty paths', () => {
        expect(isDomain('', EPathType.EPathTypeDir)).toBe(false);
    });

    it('should return false for root path', () => {
        expect(isDomain('/', EPathType.EPathTypeDir)).toBe(false);
    });
});

File: src/containers/Tenant/Schema/SchemaTree/SchemaTree.tsx [hunk 2]
```diff
@@ -110,14 +110,18 @@ export function SchemaTree(props: SchemaTreeProps) {
...
+                    {
+                        setActivePath: onActivePathUpdate,
+                        updateQueryExecutionSettings: (settings) =>
+                            setQueryExecutionSettings({...querySettings, ...settings}),
+                        showCreateDirectoryDialog: createDirectoryFeatureAvailable
+                            ? handleOpenCreateDirectoryDialog
+                            : undefined,
+                    },
+                    rootPath,
+                )}
```
Comment (artemmufazalov) [inline src/containers/Tenant/Schema/SchemaTree/SchemaTree.tsx:124]: It would be good, if you moved this to some `const` outside of jsx

**Review by artemmufazalov — APPROVED** [review]

**Pull request merged**